org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/wizard/welcome/WelcomePageConfigurator.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2015 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Simon Scholz <simon.scholz@vogella.com> - Bug 473545
+ */
+
+package org.eclipse.buildship.ui.wizard.welcome;
+
+import java.util.Collection;
+
+/**
+ * This interface is used to configure the {@link GradleWelcomeWizardPage}.
+ *
+ * @see GradleWelcomeWizardPage
+ */
+public interface WelcomePageConfigurator {
```

Comment:
I find the name configurator confusing here. This interface does not configure anything, it describes the content of a page.